Dockerfile

```diff
@@ -3,12 +3,7 @@ FROM amazonlinux:2
 WORKDIR /
 COPY . /graph-explorer/
 WORKDIR /graph-explorer
-RUN yum install -y curl
-RUN curl -sL https://rpm.nodesource.com/setup_16.x | bash -
-RUN yum install -y nodejs
-RUN yum install -y openssl
-RUN npm install -g pnpm
-RUN pnpm install
+RUN curl -sL https://rpm.nodesource.com/setup_16.x | bash - && yum install -y nodejs openssl && npm install -g pnpm && pnpm install && rm -rf /var/cache/yum
```

Comment:
I would clarify with a comment that the intention of putting all the install in one command is to reduce the size of the final image.

Reply:
Thanks, @srondelli - Done